config.yaml

```diff
@@ -29,3 +29,10 @@ options:
     type: string
     default: 192.168.252.1
     description: Gateway IP address to the Access Network.
+  external-upf-hostname:
+    type: string
+    description: |-
+      Externally accessible FQDN for the UPF.
+      If not provided, this will default to the LoadBalancer Service hostname
+      if available. If that is not available, it will default to the internal
+      Kubernetes FQDN of the service.
```

Comment:
```suggestion
      Externally accessible FQDN for the UPF.
      If not provided, it will default to the LoadBalancer Service hostname. If that is not available, it will default to the internal
      Kubernetes FQDN of the service.
```

Reply:
Not a review comment, but a question to help me understand.
Why is this a config option? how does the operator (a person?) get this info and pass it? Is not accessible in the code?

Reply:
The config option allows you to provide an elegant DNS hostname. For instance: if you use EKS and you request a LB service, you'll get an external hostname generated randomly and impossible to remember. You can map it to a nice DNS entry, but your K8s cluster nor the charm won't know about it. This config option solves that problem.